docs/docs/CHANGELOG.md

```diff
@@ -1,5 +1,27 @@
 # Changelog
 
+## v0.8.0
+
+To see a complete list of changes [see the diff](https://github.com/pomerium/pomerium/compare/v0.7.0...v0.8.0).
+
+### New
+
+- cryptutil: add automatic certificate management @desimone [GH-644]
+- implement path-based route matching @calebdoxsey [GH-615]
+- internal/identity: implement github provider support @Lumexralph [GH-582]
+
+### Changes
```

Comment:
Forward auth support for the claim headers is in this release as well.  Not sure if that goes under changes or new, though.